frontend/packages/dev-console/src/components/import/deploy-image/SearchField.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+
+import { k8sCreate } from '@console/internal/module/k8s';
+import { ImageStreamImportsModel } from '@console/internal/models';
+import { useFormikContext, FormikValues } from 'formik';
```

Comment:
I thought that would have been fixed by the linter or prettier. I know we had it for OSIO. I never worry about it since it should be auto fixed if it is important.

I have fixed it.